xml/GLIDENumber.xml

```diff
@@ -152,7 +194,7 @@
             <codeforiati:date>2022-09-01</codeforiati:date>
             <codeforiati:glide-serial>2022-000341</codeforiati:glide-serial>
         </codelist-item>
-        <codelist-item status="active">
+        <codelist-item status="withdrawn">
```

Comment:
This has been replaced by [EP-2022-000341-PAK](https://glidenumber.net/glide/public/search/details.jsp?glide=22710).